lib/queryBuilder/QueryBuilder.js

```diff
@@ -442,6 +442,10 @@ class QueryBuilder extends QueryBuilderBase {
     return this.toString();
   }
 
+  timeout(ms, options) {
+    return this.build().timeout(ms, options);
+  }
```

Comment:
This doesn't work as it's supposed to. This simply creates a knex query builder and returns it. If that query is executed, most of objection's code never gets executed. It may seem to work in some simple cases. See how the other knex methods are implemented and copy that. For example see the basic `where` method.